### DISCUSSION TITLE
Socket Hang Up Error on Serverless Function

### DISCUSSION THREAD

**Discussion by wsepesi** [vercel/community#2042] [Help]
### Summary

Hello,

I'm running a project on a hobby plan that started throwing a socket hang up error around an hour ago on all my serverless functions set up with pages/api. I was alerted to the issue after 20-30 minutes by another friend using the site, and after redeploying the project it started working again. I assume this is some sort of transient issue, but I'd love to get confirmation if there was something wrong on my end. As far as I know, no other errors were logged to the project than the one attached below, and I don't think I should have exceeded any rate limiting on the hobby plan.

I haven't made a support request before here, so let me know if there's any other information I can provide to help.

Thanks!

### Additional information

Invoke Error 	{"errorType":"Error","errorMessage":"socket hang up","code":"ECONNRESET","stack":["Error: socket hang up","    at connResetException (node:internal/errors:718:14)","    at Socket.socketOnEnd (node:_http_client:520:23)","    at Socket.emit (node:events:525:35)","    at endReadableNT (node:internal/streams/readable:1359:12)","    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)"]}

Request ID: 24hxv-1681230983179-528f9c37acb6

**Comment by amyegan** [comment]
I'm glad you got it to work after redeploying! To help me investigate, please provide as much of the following information as possible:

- A [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) repository that can be shared publicly as a git repository.
- The URL or ID of the deployment that threw this error.
- The steps required to produce the error
- Any additional information that you feel may be useful for debugging.
